tailwarden/komiser
# Feature/tech 1656

## Solution

Introduce feedback widget at Komiser dashboard

## Changes Made

- Add feedback widget
- Allow user to report an issue by capturing current screen or uploading a file
- Allow user to share their feedback

## How to Test

Check out branch and define `DISCORD_FEEDBACK_URL` when starting the backend wich the discord hook URI. Then start the frontend and click on "Leave feedback" at the top right.

## Video

https://github.com/tailwarden/komiser/assets/320272/db2ce490-42d3-46e2-90e9-f955beeb20cb

## Screenshots

![Screenshot 2023-10-10 at 17 31 46](https://github.com/tailwarden/komiser/assets/320272/af135b5c-2480-4f6a-beef-be7b656aacfb)
![Screenshot 2023-10-10 at 17 31 57](https://github.com/tailwarden/komiser/assets/320272/9e074570-0d0d-48ba-adda-61768394dec3)
![Screenshot 2023-10-10 at 17 33 20](https://github.com/tailwarden/komiser/assets/320272/55d34a60-2a88-4d13-b9c3-ba5aeaeb6c11)
![Screenshot 2023-10-10 at 17 33 23](https://github.com/tailwarden/komiser/assets/320272/b521a12d-3f5b-4377-a35f-efb4ecdfeaa5)
![Untitled](https://github.com/tailwarden/komiser/assets/320272/5cebfeed-2bb5-4e1a-b899-8845777cb07b)


## Notes
Feedback type is still to be added

## Checklist


## Reviewers

@[username of the reviewer]
